report/50_Conclusion.md

```diff
@@ -1,7 +1,11 @@
 ## Conclusion
 
-During the course of this project, the group has taken the legacy version of the MiniTwit application and refactored it into a modern ASP.NET application, which has made the application much more maintainable. Furthermore, several tools and techniques have been used to further enhance the project according to DevOps principles. This has benefitted the systems in many ways, which has been discussed in this report, and ultimately led to a more robust and transparent system with decreased lead times for new features. 
+During the course of this project, the group has taken the legacy version of the MiniTwit application and refactored it into a modern ASP.NET application, which has made the application much more maintainable. Furthermore, several tools and techniques have been used to further enhance the project according to DevOps principles. This has benefitted the systems in many ways, as layed out in this report, and ultimately led to a more robust and transparent system with decreased lead times for new features. 
 
-While working on the project it has become clear that while great benefits can be gained by adopting a DevOps mindset, it is also a lot of work. If the techniques discussed in this report were utilized more broadly in the industry, both for web development projects and otherwise, being a developer would likely be a less stressful job. But it is also important for each team to consider what parts will provide the most value, as there is famously no silver bullet and because each DevOps practice can require a significant time effort for it to be implemented properly. This is especially true when considering adding new technology to a project, as buzzwords and hype can often blind developers into grabbing a solution and then go looking for a problem. That is, developers should be careful not to focus too much on tools instead of the overall philosophy of DevOps.
+While working on the project it has become clear that while great benefits can be gained by adopting a DevOps mindset, it is also a lot of work. 
```

Comment:
Having this sentence but not expanding on it is pretty weird.

Reply:
Hvilken sætning, det er lidt svært at se?

Reply:
While working on the project it has become clear that while great benefits can be gained by adopting a DevOps mindset, it is also a lot of work.